Products/CMFPlone/interfaces/controlpanel.py

```diff
@@ -1457,6 +1458,13 @@ class IUserGroupsSettingsSchema(Interface):
     )
 
 
+def validate_twitter_username(value):
+    if value and value.startswith('@'):
+        raise Invalid(
+            u'Twitter username should not include the "@" prefix character.')
```

Comment:
no, because that's not what a validator is supposed to do: a validator must validate, not change the value of an entry; also, what happens if the user adds 2 leading "@" chars? then you will have a lot more logic on it.